airflow/models/dag.py

```diff
@@ -1223,7 +1244,7 @@ def clear(
                     include_subdags=include_subdags,
                     include_parentdag=False,
                     dag_run_state=dag_run_state,
-                    get_tis=True,
+                    get_ti_keys=True,
```

Comment:
So I'm a little bit confused as to why we had to change this method in this PR at all.

The only place this is called with `get_ti_keys` is in this recursive use here.

So I guess what I'm asking is what in this deferrable PR caused a change in behaviour to the clear function necessitating this change?

Reply:
Oh, this was caused by adding the JSON column to TI right?

Reply:
Yup, the JSON column meant the UNIONs no longer worked, since UNION compares every single field in the query for equality.